quickwit/quickwit-control-plane/Cargo.toml

```diff
@@ -13,34 +13,42 @@ documentation = "https://quickwit.io/docs/"
 anyhow = { workspace = true }
 async-trait = { workspace = true }
 chitchat = { workspace = true }
+dyn-clone = { workspace = true }
 itertools = { workspace = true }
+mockall = { workspace = true, optional = true }
```

Comment:
move mockall in dev-dependencies

Reply:
I think it's correct to add `mockall` as an optional dep if the feature `testsuite` depends on it.